satpy/readers/seviri_l1b_nc.py

```diff
@@ -159,49 +166,44 @@ def get_area_def(self, dataset_id):
         return area
 
     def get_area_extent(self, dsid):
-        if dsid.name != 'HRV':
-
-            # following calculations assume grid origin is south-east corner
-            # section 7.2.4 of MSG Level 1.5 Image Data Format Description
```

Comment:
please add the links to the relevant documentation in the docstring (see `seviri_l1b_native`). check that the links still work, too.

Reply:
done